daemon/algod/api/algod.oas2.json

```diff
@@ -3973,6 +4046,61 @@
           }
         }
       }
+    },
+    "SimulateUnnamedResourcesAccessed": {
+      "description": "If unnamed resource access is allowed, these are resources that were accessed by this group that would normally have caused failure, but were allowed in simulation. Depending on where this object is in the response, it either contains the unnamed resources that were accessed by this group and could benefit from group resource sharing (if this is a field in SimulateTransactionGroupResult), or the unnamed resources that were accessed by each transaction which cannot benefit from group sharing (if this is a field in SimulateTransactionResult). In order to make this group valid for actual submission, resources that can benefit from group sharing can be made available by any transaction of the group; otherwise, resources must be placed in the same transaction which accessed them.",
```

Comment:
I like this a lot better, and I'm willing to approve. I have two small suggestions.

Can we simply remove "If unnamed resource access is allowed, these are"?  These objects only come back if you have that enabled, right?  We don't have to say it here again.

Can we avoid the phrase "could benefit fro "?  I find that phrase really confusing - it puts me on my guard like something very subtle is being conveyed like "you could make these accesses a bit better by...."

I'm not sure I have a different phrase that is a perfect substitute but something along the lines of "could be addressed (or fixed?)"

To me, "addressed" or "fixed" is a "binary" thing - it was b ad, now it's good.  While "could benefit" seems analog (and unsure) - you _could_ make things "better".

Reply:
Good points. Here's my revised description from edc792567:

> These are resources that were accessed by this group that would normally have caused failure, but were allowed in simulation. Depending on where this object is in the response, the unnamed resources it contains may or may not qualify for group resource sharing. If this is a field in SimulateTransactionGroupResult, the resources do qualify, but if this is a field in SimulateTransactionResult, they do not qualify. In order to make this group valid for actual submission, resources that qualify for group sharing can be made available by any transaction of the group; otherwise, resources must be placed in the same transaction which accessed them.